crates/turborepo-repository/src/package_graph/mod.rs

```diff
@@ -30,6 +30,21 @@ pub struct PackageGraph {
     lockfile: Option<Box<dyn Lockfile>>,
 }
 
+#[derive(Eq, PartialEq, Hash)]
+pub struct WorkspacePackage {
+    pub name: WorkspaceName,
+    pub path: AnchoredSystemPathBuf,
+}
+
+impl WorkspacePackage {
+    pub fn root() -> Self {
+        Self {
+            name: WorkspaceName::Root,
+            path: AnchoredSystemPathBuf::from_raw("").unwrap(),
```

Comment:
Tiny nit, `AnchoredSystemPathBuf` implements `Default` so you can do this
```suggestion
            path: AnchoredSystemPathBuf::default(),
```